lisa/util/commonlib.py

```diff
@@ -0,0 +1,13 @@
+from assertpy import assert_that
+
+from pathlib import Path
+
+from lisa import Environment, Node
+from lisa.executable import CustomScript, CustomScriptBuilder
+from lisa.tools import rpm, wget
+
+def install_lis_rpms(
```

Comment:
It's not an util for all platforms. Can you create a virtual tool in `lisa\sut_orchestrator\azure\tools.py`? It likes `VmGeneration`.